### ISSUE TITLE
1.7.10 Daybreakers Mod Pack: Minemenu will not use items on a server

### ISSUE THREAD

**Issue by Twinkiepro** [dmillerw/MineMenu#28]
I understand this version is very old but I would appreciate any help I can get.

A couple friends and I use this modpack on a private server we have set up. The issue we have is that this mod is near unusable as using items is in your inventory is easily the most useful part of this mod for us.

In singleplayer, the mod works as intended for everything, yet when on the server the "use" function does not work, or at least process. Generally you open the menu and click to activate, which closes the menu and performs its function. However this bug does neither of those. MineMenu remains open and does not perform its function when pertaining to the use command.

tl;dr: Use command does not work in a server environment but all other functions do.

I can try to provide logs given the directions of where to get them. 
Here is a small clip demonstrating the issue:
https://youtu.be/0qTD8-kiMxM

**Comment by GirafiStudios** [comment]
Thanks for your report!
The use function never really worked for 1.7.10, so nothing you can do on your end.
To fix it, would require to rewrite big parts of the mod, which I have no intention of doing. Especially since the mod have not been worked on for 1.7.10 in almost 4 years.
